packages/analytics-controller/src/AnalyticsController.ts

```diff
@@ -30,69 +33,62 @@ export const controllerName = 'AnalyticsController';
  */
 export type AnalyticsControllerState = {
   /**
-   * Whether analytics tracking is enabled
-   */
-  enabled: boolean;
-
-  /**
-   * Whether the user has opted in to analytics
+   * Whether the user has opted in to analytics.
    */
   optedIn: boolean;
 
   /**
-   * User's UUIDv4 analytics identifier
+   * User's UUIDv4 analytics identifier.
+   * This is an identity (unique per user), not a preference.
+   * Must be provided by the platform - the controller does not generate it.
    */
   analyticsId: string;
 };
 
+/**
+ * Returns default values for AnalyticsController state.
+ *
+ * Note: analyticsId is NOT included - it's an identity that must be
```

Comment:
Interesting. Do we ever expect this to change? Maybe it should not be a state property?